src/plugins/ForwardMerger/forward_merger.ts

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) 2024, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { OpsBotPlugin } from "../../plugin";
+import { PayloadRepository } from "../../types";
+import { isVersionedBranch, getVersionFromBranch } from "../../shared";
+import { basename } from "path";
+import { Context } from "probot";
+import { PushEventsType } from ".";
+
+export class ForwardMerger extends OpsBotPlugin {
+  context: Context;
+  currentBranch: string;
+  repo: PayloadRepository;
+
+  constructor(
+    context: Context,
+    private payload: Context<PushEventsType>["payload"]
```

Comment:
```suggestion
    private payload: Context<"push">["payload"]
```

This can be updated.